pma_api/__init__.py

```diff
@@ -37,6 +37,7 @@ def create_app(config_name):
     from .api_1_0 import api as api_1_0_blueprint
     app.register_blueprint(api_1_0_blueprint, url_prefix='/v1')
 
-    app.add_url_rule('/', view_func=lambda: 'To be implemented.')
+    app.add_url_rule('/', view_func=lambda: jsonify({
```

Comment:
This one is supposed to be a stand-in for the documentation. Right? So the tests should not rely on this one to be JSON.

Reply:
Correct, that is stand-in for the documentation. However one advantage of the test setup that I've created is that it is dynamic. The test imports the Flask object and iterates through all routes attached (`Flask.url_map`).

There are a few solutions to this:
- (a) Make the route tests explicit rather than dynamic. (I don't recommend)
- (b) Make just the test that checks that responses return JSON explicit rather than dynamic.
- (c) Remove the JSON response test.
- (d) Add known exception cases of an endpoint returning something other than JSON.

Maybe there is some other solution as well, but I am recommending (d) and that is what I will implement in the next PR unless otherwise specified.

Reply:
I agree with having exceptions. (d) sounds like a good option.

Reply:
Will do!